patch-package
--- a/node_modules/cucumber-mink/src/mink.js
+++ b/node_modules/cucumber-mink/src/mink.js
@@ -16,6 +16,7 @@ const DEFAULT_CONFIG = {
   },
   headless: process.env.RUN_HEADLESS !== '0',
   devtools: process.env.RUN_DEVTOOLS === '1',
+  noSandbox: process.env.NO_SANDBOX === '1',
 };
 
 function gherkin(cucumber) {
@@ -40,6 +41,7 @@ Mink.prototype.setup = async function () {
   this.browser = await puppeteer.launch({
     headless: this.config.headless && !this.config.devtools,
     devtools: this.config.devtools,
+    args: (this.config.noSandbox) ? ['--no-sandbox'] : [],
   });
   this.page = await this.browser.newPage();
   return this.page.setViewport(this.config.viewport);
